api_tests/src/configure_jasmine.ts

```diff
@@ -0,0 +1,5 @@
+jasmine.getEnv().addReporter({
+    specStarted: (result) =>
+        // @ts-ignore
```

Comment:
Can we get typings for this instead of using `ts-ignore`?

Reply:
Maybe you can help me figure this one out. 

I now realise that I'm using the `jasmine` namespace to set and read a value which wasn't there in the first place i.e. there is no type because I'm creating that "field".

Would it make more sense to save this to the global context of the e2e test suite?

Reply:
> Would it make more sense to save this to the global context of the e2e test suite?

I guess not because this global context can be accessed by two tests running concurrently.

Reply:
I'm just gonna leave this one as an optional follow-up PR. Had a look at merging namespaces to extend the one provided by `jest` with the type we need here, but there's probably a much easier approach that I'm missing!

Reply:
> I guess not because this global context can be accessed by two tests running concurrently.

The test environment provides a unique instance of global to every test so you can safely set things there!

Reply:
But yes, we can do this as a follow-up. I don't have time at the moment so you could write a short issue please? :)